test/unit/annotation_spec.js

```diff
@@ -4257,6 +4257,46 @@ describe("annotation", function () {
       ]);
     });
 
+    it("should update an existing annotation", async function () {
```

Comment:
In order to get clearer output in the logs, we should probably include the annotation-type in the test name.
```suggestion
    it("should update an existing FreeText annotation", async function () {
```